.github/actions/import/action.yml

```
@@ -15,15 +15,32 @@ inputs:
   management_key:
     description: 'The management key for the Descope project'
     required: true
+  secrets:
+    description: 'An optional JSON value with additional secrets to inject into the import data'
+    default: '[]'
   files_path:
     description: 'The path to the import files (i.e., the directory containing the environment.json file)'
     default: '.'
+  validate_only:
+    description: 'Whether to only perform the validate step'
+    type: boolean
+  import_only:
+    description: 'Whether to only perform the import step'
+    type: boolean
   descopecli_version:
     description: 'The version of the descopecli tool to use'
     default: 'v0.1.0'
   descope_base_url:
     description: 'The base URL used to communicate with Descope servers (for troubleshooting purposes)'
 
+outputs:
+  failure_file:
+    description: 'The path to a file with a list of failures, or empty if there were none'
+    value: ${{ steps.validate.outputs.failure_file }}
+  secrets_file:
```

```suggestion
  missing_secrets_file:
```

`secrets_file` sounds like something one does not want to export